SPECS/libaec/libaec.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Adaptive Entropy Coding library
 Name:           libaec
-Version:        1.0.4
-Release:        5%{?dist}
+Version:        1.0.6
```

Comment:
version 1.1.2 is also available: https://gitlab.dkrz.de/k202009/libaec/-/archive/v1.1.2/libaec-v1.1.2.tar.gz